llvm/lib/Target/LoongArch/LoongArchExpandPseudoInsts.cpp

```diff
@@ -167,6 +169,39 @@ bool LoongArchPreRAExpandPseudo::expandMI(
   case LoongArch::PseudoTAIL_MEDIUM:
   case LoongArch::PseudoTAIL_LARGE:
     return expandFunctionCALL(MBB, MBBI, NextMBBI, /*IsTailCall=*/true);
+  case LoongArch::PseudoBRIND: {
+    // If the PseudoBRIND is used to table jump, then emit a label to annotate
```

Comment:
Move these statements into a new method function?

Reply:
Thanks, I will do that.